.fontello

```diff
@@ -0,0 +1 @@
+a050d58fb8b988cfa315f567a1cc814d
```

Comment:
What is this hash for? Hopefully not a secret key.

Reply:
everything in this folder is now generated by the makefile script - so no we dont use a bunch of it but it would be wayyy more trouble than its worth to try and mess around with how much we keep :) so yes its a hash in this case

Reply:
oh shit sorry i didnt see where this file was located - my bad i merged a bit hastily

ill raise a PR to flush just this

Reply:
Yeah these code generation things are a bit of a nightmare for us humans. It's no big deal, just annoying and potentially confusing to have unused stuff lying around.